### DISCUSSION TITLE
Outline view in a separate pane

### DISCUSSION THREAD

**Discussion by scottrmead** [notaapp/talk#265] [Feature Requests] [answered]
I'd like an outline pane similar to the Caret outline pane for navigating headings in documents.

**Comment by scottrmead** [comment, marked as answer]
Nevermine.  My initial search was poor.  FOund the planned feature request here: https://github.com/notaapp/talk/discussions/259

**Reply by astoilkov** [reply]
Great. I will delete this discussion as it's confusing when you explore the other feature requests.
